### ISSUE TITLE
/prepare.sh: patch does not apply

### ISSUE THREAD

**Issue by ioletsgo** [NyanCatTW1/robloxWineBuildGuide#21]
I tried following the steps (Arch Linux) but while trying to apply the patch I got this error
```
[ioletsgo@IdeaPad Documents]$ git clone https://github.com/Frogging-Family/wine-tkg-git.git
Cloning into 'wine-tkg-git'...
remote: Enumerating objects: 14491, done.
remote: Counting objects: 100% (1152/1152), done.
remote: Compressing objects: 100% (553/553), done.
remote: Total 14491 (delta 630), reused 1010 (delta 589), pack-reused 13339
Receiving objects: 100% (14491/14491), 12.63 MiB | 1.85 MiB/s, done.
Resolving deltas: 100% (11001/11001), done.
[ioletsgo@IdeaPad Documents]$ cd wine-tkg-git/
[ioletsgo@IdeaPad wine-tkg-git]$ curl https://raw.githubusercontent.com/e666666/robloxWineBuildGuide/main/roblox-wine-staging-v2.3.patch --output roblox-wine-staging-v2.3.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3805  100  3805    0     0  12429      0 --:--:-- --:--:-- --:--:-- 12434
[ioletsgo@IdeaPad wine-tkg-git]$ git apply roblox-wine-staging-v2.3.patch 
error: patch failed: wine-tkg-git/wine-tkg-scripts/prepare.sh:666
error: wine-tkg-git/wine-tkg-scripts/prepare.sh: patch does not apply
[ioletsgo@IdeaPad wine-tkg-git]$ 
```

**Comment by NyanCatTW1** [comment]
Fixed by #20 hopefully

**Comment by ioletsgo** [comment]
I got this error while building
`[1m[34m  ->(B[m[1m Cleaning wine-staging source code tree...(B[m
HEAD is now at c9652a68 Rebase against 29dd84443976005a8ec713dd1c75458e27346591.
[1m[34m  ->(B[m[1m Cleaning wine source code tree...(B[m
HEAD is now at 29dd8444397 dbghelp: Use remap table from PDB hash stream.

af8ed02b572081206be6c505261f5f2e98a8053c reverted (hotfix)

313f75a0afc108db904aae4885da1f3783cd34c5 reverted (hotfix)
Auto-merging dlls/user32/menu.c
CONFLICT (content): Merge conflict in dlls/user32/menu.c
Auto-merging dlls/win32u/menu.c
CONFLICT (content): Merge conflict in dlls/win32u/menu.c
Auto-merging include/ntuser.h`
